src/basic/renderers/choice.ts

```diff
@@ -53,20 +53,11 @@ class ChoiceCallbackRenderer implements DestroyableCallbackRenderer, FocusableCa
    */
   public render = (): HTMLDivElement => {
     const defaultIndex = this.callback.getInputValue() as number;
-    const formGroup = el<HTMLDivElement>('div', `fr-callback-${this.index} form-group`);
+    const formGroup = el<HTMLDivElement>('div', `fr-callback-${this.index} form-group mb-3`);
     const id = `fr-callback-${this.index}`;
 
-    // Add the prompt
-    const prompt = this.callback.getPrompt();
-    if (prompt) {
-      const label = el<HTMLLabelElement>('label');
-      label.innerHTML = prompt;
-      label.htmlFor = id;
-      formGroup.appendChild(label);
-    }
-
     // Add the dropdown
-    this.input = el<HTMLSelectElement>('select', 'form-control');
+    this.input = el<HTMLSelectElement>('select', 'form-select');
```

Comment:
Needed to change this to provide the appropriate styling without `ui-design` library.